### ISSUE TITLE
win_find impossibile to have results from volume path

### ISSUE THREAD

**Issue by NaughtyTommy91** [ansible-collections/ansible.windows#103]
##### SUMMARY

Hello All,
during this days i tryed with my teams to use the win_find module for search inside volumes on a Windows server (like C:, D:, E:) different type of file. During the scan the module show this error:
"win_find failed to check some files these files were ignored"

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME

!component =lib/ansible/modules/windows/win_find.ps1

##### ANSIBLE VERSION

```paste below
ansible 2.9.5
```

##### CONFIGURATION

```paste below
DEFAULT_BECOME(/home/ansible/ansible.cfg) = False
DEFAULT_BECOME_ASK_PASS(/home/ansible/ansible.cfg) = False
DEFAULT_BECOME_METHOD(/home/ansible/ansible.cfg) = sudo
DEFAULT_BECOME_USER(/home/ansible/ansible.cfg) = root
DEFAULT_HOST_LIST(/home/ansible/ansible.cfg) = ['/home/ansible/inventory']
DEFAULT_REMOTE_USER(/home/ansible/ansible.cfg) = ansible
```

##### OS / ENVIRONMENT

Red Hat 8.2.1-3

##### STEPS TO REPRODUCE

`` 
If i try with this code, for example:

- name: Find the installation path and files
   win_find:
     paths: 'E:\'
     patterns: ['.sh','.bin','.pl','.ear','.SH','.BIN','.PL','.EAR','.exe','.sys','.com','.ocx']
     recurse: yes
     file_type: file
   register: data`

The results was "win_find failed to check some files these files were ignored"

If i try:

- name: Find the installation path and files
   win_find:
     paths: 'E:\temp'
     patterns: ['.sh','.bin','.pl','.ear','.SH','.BIN','.PL','.EAR','.exe','.sys','.com','.ocx']
     recurse: yes
     file_type: file
   register: data`

In this case no problem

```yaml
- name: Find the installation path and files
   win_find:
     paths: 'E:\'
     patterns: ['.sh','.bin','.pl','.ear','.SH','.BIN','.PL','.EAR','.exe','.sys','.com','.ocx']
     recurse: yes
     file_type: file
   register: data`
```

##### ACTUAL RESULTS

"win_find failed to check some files these files were ignored"

```paste below

```

**Comment by jborean93** [comment]
See https://github.com/ansible-collections/ansible.windows/issues/101#issuecomment-689087530.

The win_find module has been rewritten for this collection (will be available in the box with Ansible 2.10). One of the bigger improvements was the speed and support for enumerating highly nested directories (like a drive root). I would try out the new version by installing the collection with

```bash
ansible-galaxy collection install ansible.windows
```

Then your task has `ansible.windows.win_find:` instead of just `win_find:`. See how that goes and if it gives you the results you are looking for.

**Comment by jborean93** [comment]
Closing as I assume the changes in the refactor has fixed this issue.
